index.html

```diff
@@ -1,12 +1,197 @@
+          <li class="bordered-link"><a class="close" href="#">CONTACT US</a></li>
+        </ul>
+      </nav>
+    </div>
+    <div class="desktop-navigation">
+      <div class="top-nav-bar">
+        <ul>
+          <li><a href="www.facebook/DatStarTechnology" class="social-icons"><i class="fa fa-facebook" aria-hidden="true"></i></a></li>
+          <li><a href="#" class="social-icons"><i class="fa fa-twitter" aria-hidden="true"></i></a></li>
+          <li><a href="t.me/datstartech3" class="social-icons"><i class="fa fa-telegram" aria-hidden="true"></i></a></li>
+          <li><a href="DatStarTechnology@gmail.com" class="social-icons"><i class="fa fa-envelope-o" aria-hidden="true"></i></a></li>
+          <li><a href="index.html" class="social-icons">My Page</a></li>
+        </ul>
+      </div>
+      <div class="header-container">
+        <div class="logo">
+          <a href="./index.html">
+            <img src="./images/logo.png" alt="DATStar Technology">
+          </a>
+        </div>
+        <nav class="desktop-nav">
+          <ul>
+            <li><a href="./about.html">About</a></li>
+            <li><a href="#service">Service</a></li>
+            <li><a href="#blog">Blogs</a></li>
+            <li><a href="#support">Support</a></li>
+            <li><a href="#news">News</a></li>
+            <li class="bordered-link"><a href="#">CONTACT US</a></li>
+          </ul>
+        </nav>
+      </div>
+    </div>
+  </header>
+  <main class="main-section-container">
+    <div class="transparent">
+      <div class="main-section">
+        <h2 class="title-intro">
+          <q>Welcome to DATStar Technology</q>
+        </h2>
+        <h1 class="title-info">Your Number One GPS Tracking and Security System</h1>
+        <div class="description">
+          <p>We offer a system that allows our clients to remotely control their assets. Our system assists the client in managing vehicle usage, 
+        fuel, tire performance, driver performance, and vehicle maintenance.</p>
+        </div>
+        <div class="office">
+          <h2>We are open from Monday ~ Saturday(12AM)</h2>
+          <p>You can reach us: <br> @Cabey Building 3rd floor, Room Number: 304</p>
+        </div>
+      </div>
+    </div>
+  </main>
+  <section class="main-service">
+    <h2 class="section-titles">Main Services</h2>
+    <hr class="section-title-separators">
+    <ul class="main-service-grid">
+      <li class="service-list">
+        <div class="service-icon">
+          <img src="./images/system.png" alt="">
+        </div>
+        <div class="service-title">
+          <h3>Top-Notch Design</h3>
+        </div>
+        <div class="service-description">
+          <p>Our tracking unit is designed with latest technology which makes it work with all kinds of vehicles.</p>
+        </div>
+      </li>
+      <li class="service-list">
+        <div class="service-icon">
+          <img src="./images/realtime.png" alt="">
+        </div>
```

Comment:
- [ ] Nice putting the hover effect on this `service-list` only one thing to improve is to put change the cursor to a pointer when hovering. 💪  💪 
![image](https://user-images.githubusercontent.com/65068771/147224339-7e6ef408-8e4d-43b0-8d59-6efd2d2748ce.png)